lib/options.js

```diff
@@ -201,7 +204,7 @@ try re-running it with ${chalk.cyan('--elmjson <path-to-elm.json>')}.`,
     forceBuild: args['force-build'],
     report: args.report === 'json' || args.report === 'ndjson' ? 'json' : null,
     reportOnOneLine: args.report === 'ndjson',
-    rulesFilter: listOfStrings(args.rules),
+    rulesFilter: listOfStrings(args.rules) || [],
```

Comment:
I think this should stay `null`, as I think we do check whether this has been set.
The Options and the different types I created in the `lib/types/*.d.ts` files have been created in a way to resemble what I did in JS, but might not always been correct. So far, the JS part seems way more correct than the types (it's for now the source of truth, though hopefully we'll find bugs with all of this!), so feel free to change the types if they seem to contradict JS.

https://github.com/jfmengels/node-elm-review/blob/df9a5107b01a8dcd8b56cc8988c10e39e7c94372/lib/result-cache.js#L83

Reply:
Yes, there are checks for it everywhere it's used. I'll let it be null then 👍